src/ObjCRuntime/Registrar.cs

```diff
@@ -1543,6 +1544,7 @@ void VerifyTypeInSDK (ref List<Exception> exceptions, TType type, ObjCMethod par
 					if (attrib.Version <= sdk)
 						break;
 
+					//todo: localize msg
 					string msg = "The type '{0}' (used as {1} {2}) is not available in {3} {4} (it was introduced in {3} {5}){6} Please build with a newer {3} SDK (usually done by using the most recent version of Xcode).";
```

Comment:
Please create an issue, that way we won't forget.

Reply:
I've added all mmp errors to this PR now including this and the other ones you commented on.